### PR TITLE
Add terms to Dance Hall / Dance School presets

### PR DESCRIPTION
**Pull request by FloEdelmann**
For **first suggestion** of #1177.

### CODE REVIEW

File: data/presets/leisure/dancing_school.json [hunk 1]
```diff
@@ -20,7 +20,10 @@
         "swing",
         "tango",
         "waltz",
-        "dance teaching"
+        "ballet",
+        "dance teaching",
+        "ballet studio",
+        "dance studio"
```
Comment (FloEdelmann) [inline data/presets/leisure/dancing_school.json:26]: Since there is an alias called "Dance Studio" here, does it make sense to add it as a term as well?

Comment (tyrasd) [reply data/presets/leisure/dancing_school.json:26]: No, it's not necessary to repeat the aliases here. Let's drop it here.

**Review by tyrasd — CHANGES_REQUESTED** [review]
the additional search terms look reasonable to me. However, the `terms` should be sorted alphabetically. Can you please change that?

File: data/presets/leisure/dancing_school.json (hunk 1 above)
Comment (FloEdelmann) [reply data/presets/leisure/dancing_school.json]: Thanks for clarifying. Removed in [`6cf3fe4`](https://github.com/openstreetmap/id-tagging-schema/pull/1182/commits/6cf3fe4611f48a3ddcfee8e1e8c2b24a6fc7d0eb).

**Review by tyrasd — APPROVED** [review]

**Pull request merged**